config/chartdir/mychart.yaml

```diff
@@ -13,6 +13,9 @@ length: 60
 # json name of x_axis data
 x_axis_name: "timestamp"
 
+# ngx-charts consumable data
+ngx: true
```

Comment:
should this be something like `ngx-area` or more general like `nested` or `multi` or something?